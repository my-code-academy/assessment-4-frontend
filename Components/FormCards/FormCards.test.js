import React from 'react';
import renderer from 'react-test-renderer';
import FormCard from './FormCard.component';


describe('FormCard', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<FormCard />);
    expect(tree).toMatchSnapshot();
  });
});
