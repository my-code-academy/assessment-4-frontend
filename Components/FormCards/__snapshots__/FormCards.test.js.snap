// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormCard should match the snapshot 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "white",
      "color": "black",
      "margin": 10,
      "opacity": 1,
      "width": 300,
    }
  }
>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 20,
        "margin": 20,
      }
    }
  />
  <View
    style={
      Object {
        "borderBottomColor": "black",
        "borderBottomWidth": 1,
      }
    }
  />
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "black",
        "fontSize": 10,
        "margin": 10,
        "textAlign": "right",
      }
    }
  >
    created At:
     
  </Text>
</View>
`;
